HousingRepairsOnlineApi/UseCases/BookAvailableAppointmentUseCase.cs

```diff
@@ -35,7 +36,13 @@ public async Task Execute(string bookingReference, string sorCode, string priori
             {
                 throw new InvalidOperationException($"No appointments returned from Appointments Gateway");
             }
-            await appointmentsGateway.BookAppointment(bookingReference, sorCode, priority, locationId, appointment.TimeOfDay.EarliestArrivalTime, appointment.TimeOfDay.LatestArrivalTime, repairDescriptionText);
+
+            var communalRepairCommentText = repairDescriptionText + " " + repairDescriptionLocation;
+            if (communalRepairCommentText.Length > 255)
```

Comment:
This code base doesn't validate the length of the description, it is performed in the Scheduling API ([here](https://github.com/Newarkandsherwood/HousingRepairsSchedulingApi/blob/main/HousingRepairsSchedulingApi/UseCases/BookAppointmentUseCase.cs#L26)).

Each scheduling may be able to handle a different max length of description (which is [yet to be made configurable](https://github.com/Newarkandsherwood/HousingRepairsSchedulingApi/pull/17#discussion_r987684717)).

Please remove this validation and rely on the Scheduling API instead.